defs/build.bzl

```diff
@@ -134,7 +137,7 @@ def _hashes_for_srcs(srcs, srcs_basenames_needing_hashes):
 # srcs is expected to be label list.
 # conditioned_srcs is a dictionary mapping conditions to label lists.
 #   It will be passed to select().
-def release_filegroup(name, srcs = None, conditioned_srcs = None, visibility = None):
+def release_filegroup(name, srcs = None, conditioned_srcs = None, tags = None, visibility = None):
```

Comment:
Is this part of another commit? I don't see where k8s_deb or pkg_rpm call this (and/or now use tags)?

Reply:
release_filegroup isn't used directly in any rules here, but it's used in a number of places over in k/k.